config/kubernetes/staging/ingress.yml

```diff
@@ -24,6 +24,9 @@ metadata:
       SecDefaultAction "phase:2,pass,log,tag:github_team=laa-crime-apply"
       SecRuleUpdateTargetById 941100-941350 !REQUEST_COOKIES:_laa_review_criminal_legal_aid_session
       SecRuleUpdateTargetById 942100 !REQUEST_COOKIES:_laa_review_criminal_legal_aid_session
+      SecRule REQUEST_URI "@beginsWith /api/events" "id:555001, phase:1, pass, t:none, nolog, chain" \
+        SecRule REQUEST_METHOD "@streq POST" "chain" \
+          SecRule REQUEST_HEADERS:Content-Type "@streq text/plain" "ctl:ruleRemoveById=920480"
```

Comment:
Wouldn't be ruleId `920420` instead of `920480`?

Reply:
Additionally maybe this is simpler?
`SecRuleUpdateTargetById 920420 !REQUEST_URI:/api/events`

Reply:
YES!